src/cross_toml.rs

```diff
@@ -69,6 +95,32 @@ impl CrossToml {
         self.get_target(target).and_then(|t| t.image.clone())
     }
 
+    /// Returns the `target.{}.dockerfile` or `target.{}.dockerfile.file` part of `Cross.toml`
```

Comment:
This always returns `target.{}.dockerfile.file`, since this is from the config file only, and `file` is not optional now. Should be:

```rust
/// Returns the `target.{}.dockerfile.file` part of `Cross.toml`
```

Reply:
wrong link, https://github.com/cross-rs/cross/blob/0ecb3279fba1572cbbc52fd33a80603248334110/src/cross_toml.rs#L34-L35